### PR TITLE
Bump version to `v0.4.5`

### PR DESCRIPTION
**Pull request by snowsignal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,70 @@
...
+- \[`pyflakes`\] Update docs to describe WAI behavior (F541) ([#11362](https://github.com/astral-sh/ruff/pull/11362))
+
+## CI
+- Add automation for updating our vendored typeshed stubs ([#11427](https://github.com/astral-sh/ruff/pull/11427))
+
+## Internal
+- Migrate `sys.rs` generation to `stdlibs` ([#11374](https://github.com/astral-sh/ruff/pull/11374))
+
+### Other changes
+- Add Python 3.13 to list of allowed Python versions ([#11411](https://github.com/astral-sh/ruff/pull/11411))
```
Comment (snowsignal) [inline CHANGELOG.md:66]: @charliermarsh Should we include this, and if so, what would be a good section for it?

Comment (charliermarsh) [reply CHANGELOG.md:66]: I think we stopped doing this

Comment (charliermarsh) [reply CHANGELOG.md]: Sorry, I thought this was on the blog post (I saw it on my phone). Lets put this under `Documentation` I guess?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
+
+- Add a note on `--preview` to the README ([#11395](https://github.com/astral-sh/ruff/pull/11395))
+- Simplify Neovim setup documentation ([#11489](https://github.com/astral-sh/ruff/pull/11489))
+- Update CONTRIBUTING.md to reflect the new parser ([#11434](https://github.com/astral-sh/ruff/pull/11434))
+- Update server documentation with new migration guide ([#11499](https://github.com/astral-sh/ruff/pull/11499))
+- \[`pycodestyle`\] Clarify motivation for `E713` and `E714` ([#11483](https://github.com/astral-sh/ruff/pull/11483))
+- \[`pyflakes`\] Update docs to describe WAI behavior (F541) ([#11362](https://github.com/astral-sh/ruff/pull/11362))
+- \[`pylint`\] Clearly indicate what is counted as a branch ([#11423](https://github.com/astral-sh/ruff/pull/11423))
+
+## CI
```
Comment (charliermarsh) [inline CHANGELOG.md:58]: Lets remove this section and "Internal" since it doesn't affect users.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,68 @@
 # Changelog
 
+## 0.4.5
+
+### Ruff's language server is now in Beta
+
+`v0.4.5` marks the official Beta release of `ruff server`, an integrated language server built into Ruff.
+`ruff server` does everything `ruff-lsp` does and more, bringing significant improvements to performance along the way.
```
Comment (charliermarsh) [inline CHANGELOG.md:8]: Can we saw something like:

> `ruff server` supports the same feature set as `ruff-lsp`, but with superior performance and no installation required. We'd love your feedback.

Just to match the language we added to the https://github.com/astral-sh/ruff-lsp README and elsewhere.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,60 @@
...
+- Sort edits prior to deduplicating in quotation fix ([#11452](https://github.com/astral-sh/ruff/pull/11452))
+- Treat escaped newline as valid sequence ([#11465](https://github.com/astral-sh/ruff/pull/11465))
+- \[`flake8-pie`\] Preserve parentheses in `unnecessary-dict-kwargs` ([#11372](https://github.com/astral-sh/ruff/pull/11372))
+- \[`pylint`\] Ignore `__slots__` with dynamic values ([#11488](https://github.com/astral-sh/ruff/pull/11488))
+- \[`pylint`\] Remove `try` body from branch counting ([#11487](https://github.com/astral-sh/ruff/pull/11487))
+- \[`refurb`\] Respect operator precedence in `FURB110` ([#11464](https://github.com/astral-sh/ruff/pull/11464))
+
+### Documentation
+
+- Add a note on `--preview` to the README ([#11395](https://github.com/astral-sh/ruff/pull/11395))
```
Comment (charliermarsh) [inline CHANGELOG.md:50]: Nit: "Add `--preview` to the README"

File: CHANGELOG.md (hunk 3 above)
Comment (charliermarsh) [reply CHANGELOG.md:8]: Lets add a second paragraph that's just like:

You can enable `ruff server` in the [VS Code extension](https://github.com/astral-sh/ruff-vscode?tab=readme-ov-file#enabling-the-rust-based-language-server) today.

What do you think?

Comment (snowsignal) [reply CHANGELOG.md:8]: Should we mention that `ruff server` has several brand-new features too?

Comment (charliermarsh) [reply CHANGELOG.md:8]: I just want there to be a call-to-action without going through the blog post.

Comment (charliermarsh) [reply CHANGELOG.md:8]: Actually:

> `ruff server` supports the same feature set as `ruff-lsp`, powering linting, formatting, and code fixes in Ruff's editor integrations -- but with superior performance and no installation required. We'd love your feedback.

Something to make clear it's not a general LSP?

Comment (snowsignal) [reply CHANGELOG.md:8]: Ah, to be clear, this was a question for the first suggestion.

Comment (snowsignal) [reply CHANGELOG.md:8]: @charliermarsh That sounds good!

**Review by charliermarsh — APPROVED** [review]

**Pull request merged**